serverless.yml

```diff
@@ -0,0 +1,53 @@
+service: ${env:SERVICE_NAME, "saf-lambda-function"}
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+  iam:
+    role:
+      path: ${env:IAM_ROLE_PATH, ""}
+      permissionsBoundary: ${env:IAM_ROLE_BOUNDARY, ""}
+      statements:
+        - Effect: "Allow"
+          Action:
+            - "s3:*"
+          Resource:
+            - "*"
+  deploymentBucket: # Custom configuration for deployment bucket is unique and put here
+    blockPublicAccess: true
+  s3: # Custom configuration for any other s3 buckets can be added in this section
+    inputBucket:
+      name: ${env:INPUT_BUCKET}
+      publicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+  environment:
+    INPUT_BUCKET: ${env:INPUT_BUCKET}
+
+functions:
+  saf:
+    timeout: ${env:TIMEOUT, 60}
+    handler: handler.saf
+    events:
+      - s3:
+          bucket: inputBucket
+          event: s3:ObjectCreated:*
```

Comment:
Test with `s3:ObjectCreated:Put`

Reply:
Test failed, the function is still being invoked more than once.

Reply:
**Uploaded 1 object:**

![Screen Shot 2022-06-09 at 12 56 12 PM](https://user-images.githubusercontent.com/11844975/172902468-5d93d769-ce64-43f4-b534-4cfd6049d303.png)

**Invocations:**

![Screen Shot 2022-06-09 at 12 56 48 PM](https://user-images.githubusercontent.com/11844975/172902563-c2ec4e4a-4f2d-4a5e-8ec1-aecf80c564aa.png)